composer.json

```diff
@@ -28,19 +28,23 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "http-interop/http-middleware": "^0.4.1",
-        "laminas/laminas-mvc": "^3.1",
+        "php": "^7.3",
+        "laminas/laminas-mvc": "^3.2",
         "laminas/laminas-psr7bridge": "^1.2.2",
-        "laminas/laminas-servicemanager": "^3.3",
-        "laminas/laminas-stratigility": "^2.2.2p2"
+        "laminas/laminas-servicemanager": "^3.4",
+        "laminas/laminas-stratigility": "^3.3",
+        "psr/http-server-handler": "^1.0",
+        "psr/http-server-middleware": "^1.0",
```

Comment:
The  latter depends on the former, so you really only need the middleware package here.

Reply:
Handler interface was used in this package code, so I list it as direct dependency. It worked well for me in preventing issues with forward compat changes bringing new major version of implicit dependency.
I think I will put this approach on TSC agenda as a QA requirement.

Reply:
https://github.com/maglnet/ComposerRequireChecker will help you with that

Reply:
And, of course, first thing I see there is Ocramius merging PRs.

Reply:
it was basically his idea, if I understood correctly